<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_selected">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>611</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Which online game is better?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_formula">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/csgo-vs-valorant.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CS:GO</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Valorant</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
